package/lib/SimpleSchema.js

```diff
@@ -8,7 +8,6 @@ import MessageBox from 'message-box';
 import MongoObject from 'mongo-object';
 import omit from 'lodash.omit';
 import pick from 'lodash.pick';
-import uniq from 'lodash.uniq';
```

Comment:
This was the last use of `lodash.uniq` package, so please `npm uninstall lodash.uniq`

Reply:
Thanks for the review, dependency removed